index.html

```diff
@@ -185,8 +184,8 @@ <h2>About me</h2>
 			</div>
 
 			<div class="experience">
-				<div class="about-card">
-					<img src="../assets/languages.png" alt="" />
+				<div class="about-card crd1">
+					<img src="../assets/languages.png" alt="Languages" />
```

Comment:
- [ ] I think the source images do not exist in your assets folder. that's why these images are not showing up. Kindly consider fixing them in all the skills cards.
your design
![re](https://user-images.githubusercontent.com/96683843/213876897-36faf078-3b2c-4402-8ab6-d14282748230.jpg)